djweb/dj_comp_hist/views.py

```diff
@@ -179,13 +179,22 @@ def browse(request):
     return render(request, 'browse.jinja2')
 
 def generate_search_facets(doc_objs):
+    """
+    Creates a dictionary with dictionaries of each facet of the given parameter. Each of the facet dictionaries contain
+    keys that are the top 10 most common instances of that given facet and values that signify the number of
+    occurrences of that specific instance.
+
```

Comment:
the way I ran this (might be useful to include in documentation)
In terminal:
```
cd djweb
python manage.py shell
```
In shell:
```
from dj_comp_hist.models import Document
doc_objs = Document.objects
from dj_comp_hist.views import generate_search_facets
generate_search_facets(doc_objs)
```